src/main/java/net/dries007/tfc/objects/potioneffects/PotionThirst.java

```diff
@@ -0,0 +1,50 @@
+package net.dries007.tfc.objects.potioneffects;
+
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.player.EntityPlayerMP;
+
+import net.dries007.tfc.api.capability.food.IFoodStatsTFC;
+
+public class PotionThirst extends PotionTFC
+{
+    public PotionThirst()
+    {
+        super(true, 0x2c86d4);
+        setPotionName("effectsTFC.thirst");
+        setIconIndex(0, 1);
+    }
+
+    @Override
+    public void performEffect(EntityLivingBase entity, int amplifier)
+    {
+        EntityPlayerMP player = null;
+        IFoodStatsTFC foodStatsTFC = null;
+        if (entity instanceof EntityPlayerMP)
+        {
+            player = (EntityPlayerMP) entity;
+            if (player.getFoodStats() instanceof IFoodStatsTFC)
+            {
+                foodStatsTFC = (IFoodStatsTFC) player.getFoodStats();
+            }
+        }
+
+        if (player != null && foodStatsTFC != null)
+        {
+            float thirst = foodStatsTFC.getThirst();
+
+            foodStatsTFC.setThirst(thirst - 0.05F * (float) (amplifier + 1));
+        }
+    }
+
+    @Override
+    public boolean isReady(int duration, int amplifier)
+    {
+        return true;
```

Comment:
Minor nitpick: I think it would be better to do like hunger and drains the thirst over time instead of instantly.

Reply:
This is exactly like hunger? It's a copy paste of the hunger debuff set with a higher default (one order higher) so you don't need a 40 modifier on it to notice the drop.